### PR TITLE
Adding linted header

### PR DESCRIPTION
**Pull request by Alogator1**

### CODE REVIEW

File: client/src/modules/MapHolder/index.js [hunk 1]
```diff
@@ -8,14 +8,13 @@ import SYMBOL_LAYOUT from './symbolLayout';
...
 import { setMap } from '../../actions/map';
-import { sidebarWidth } from '../Sidebar/styleConstants';
 
 const useStyles = makeStyles({
   mapOuterStyle: {
     position: 'relative',
     height: '100vh',
     overflow: 'hidden',
-    width: `calc(100vw - ${sidebarWidth})`
+    width: 'calc(100vw)',
```
Comment (OrestSat) [inline client/src/modules/MapHolder/index.js]: calc is not needed

File: client/src/modules/MapHolder/index.js [hunk 2]
```diff
@@ -55,26 +54,29 @@ const MapHolder = ({
   };
 
   return (
-    <Map
+    <div className={classes.mapOuterStyle}>
```
Comment (OrestSat) [inline client/src/modules/MapHolder/index.js]: Why do you use same style for map and map wrapper?

File: client/src/modules/Sidebar/components/Filter/index.js [hunk 3]
```diff
@@ -1,23 +1,66 @@
...
+
   return (
-    <div>
-      <input
-        style={{
-          width: '100%',
-          marginBottom: '1rem',
-          padding: '0.5rem'
+    <div className={classes.searchWrapper}>
+      <InputBase
```
Comment (OrestSat) [inline client/src/modules/Sidebar/components/Filter/index.js]: Please Use formik input

File: client/src/modules/Sidebar/components/Header/index.js [hunk 4]
```diff
@@ -1,7 +1,105 @@
...
+
+  const handleChange = () => {
+    setChecked(prev => !prev);
+  };
+
+  const changeSidebarVisibility = () => {
+    setVisible(prev => !prev);
+  };
+
+  const myClasses = useStyles();
```
Comment (OrestSat) [inline client/src/modules/Sidebar/components/Header/index.js]: Please use classes to follow project rules

File: client/src/modules/Sidebar/components/Header/index.js [hunk 5]
```diff
@@ -1,7 +1,105 @@
...
+
+  return (
+    <div className={myClasses.headerWrapper}>
+      <div className={myClasses.headerContainer}>
+        <img src="https://images.vectorhq.com/images/previews/cca/defibrillator-logo-121414.png" className={myClasses.headerLogo} alt='logo' />
```
Comment (OrestSat) [inline client/src/modules/Sidebar/components/Header/index.js]: Move url to const

File: client/src/modules/Sidebar/index.js [hunk 6]
```diff
@@ -1,31 +1,33 @@
...
-import { sidebarWidth } from './styleConstants';
 
 const useStyles = makeStyles({
   sidebarStyle: {
     display: 'flex',
     flexDirection: 'column',
     justifyContent: 'space-between',
-    width: sidebarWidth,
-    padding: 20,
+    width: '360px',
```
Comment (OrestSat) [inline client/src/modules/Sidebar/index.js]: Why do you remove sidebarWidth const?

File: client/src/modules/Sidebar/index.js [hunk 7]
```diff
@@ -1,31 +1,33 @@
...
+    padding: '20px',
```
Comment (OrestSat) [inline client/src/modules/Sidebar/index.js]: 'px' is not needed

File: client/src/modules/MapHolder/index.js [hunk 8]
```diff
@@ -1,6 +1,9 @@
 import React, { useState } from 'react';
 import { connect } from 'react-redux';
 import { makeStyles } from '@material-ui/core/styles';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import Button from '@material-ui/core/Button';
+import { GeoJSONLayer } from 'react-mapbox-gl';
```
Comment (oskovbasiuk) [inline client/src/modules/MapHolder/index.js]: is not used in file

**Pull request merged**